README.md

```diff
@@ -28,61 +28,64 @@ original everware team consists of
 [@anaderi](https://github.com/anaderi), and
 [@AxelVoitier](https://github.com/AxelVoitier).
 
+
 ## Documentation
 
 We have some documentation on the different ways to use `everware` and
 [getting
 started](https://github.com/everware/everware/wiki/Getting-Started).
 
+
 ## Try it out
 
 In order to deploy your own `everware` instance, you have to:
 
- - Install the newest git version of `jupyterhub`: https://github.com/jupyter/jupyterhub. Double check with their [README.md](https://github.com/jupyter/jupyterhub/blob/master/README.md)
-```
-    sudo apt-get install npm nodejs-legacy
-    sudo npm install -g configurable-http-proxy
-    git clone https://github.com/jupyter/jupyterhub.git
-    cd jupyterhub
-    pip3 install -r dev-requirements.txt -e .
+ - create and activate a `conda` environment with python 3.5
 ```
- - Install the newest git version of `dockerspawner`: https://github.com/jupyter/dockerspawner. Double check with their [README.md](https://github.com/jupyter/dockerspawner/blob/master/README.md)
+    conda create -n everware python=3.5
 ```
-    git clone https://github.com/jupyter/dockerspawner.git
-    cd dockerspawner
-    pip3 install -e .
-```
- - Clone this repo
+ - Clone this repo and install `everware`
 ```
     git clone https://github.com/everware/everware
     cd everware
+    make install
 ```
- - [Create a Github OAuth application](https://github.com/settings/applications/new) with URL `http://localhost:8000/` and callback URL `http://localhost:8000/hub/oauth_callback`
- - Enter you OAuth information into `env.sh` and `source env.sh`
- - Install everware
+ - [Create a Github OAuth application](https://github.com/settings/applications/new)
+   with URL `http://localhost:8000/` and callback URL
+   `http://localhost:8000/hub/oauth_callback`
+ - Enter you OAuth information into `env.sh` and `source env.sh`. You will
+   have to set these environment variables every time before running `everware`
+
+ - If you are on **Mac OS** make sure the VM in which docker runs is
```

Comment:
As I have no knowledge at all about windows and also can't test it I created an issue instead: #92